### ISSUE TITLE
@AutoValue generated source failed to compile with custom annotation with string value in Eclipse

### ISSUE THREAD

**Issue by laec** [google/auto#219]
Environment: Eclipse + m2e-apt

The follow class (simplified):
```
@AutoValue
abstract class Image {

  @CustomAnnotation("Width")
  abstract int width();

}
```

Will generate (simplified):
```
final class AutoValue_Image {

  @CustomAnnotation(value=Width) // Failed to compile as "Width" aren't in double quotes
  @Override
  int width() {
    return width;
  }

}

```

**Comment by eamonnmcmanus** [comment]
Looks like a dup of #168. That has been fixed in 1.1-SNAPSHOT but the bug is still present in 1.0.

**Comment by cjp39** [comment]
Any idea when 1.1 will be cut? This bug is causing us a lot of pain :(
